compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -127,6 +127,19 @@ void quantize_and_verify_with_wrong_type(luci::test::TestIOGraph *g, Type quanti
   verifier.verify(g->g());
 }
 
+void quantize_and_verify_with_wrong_type(luci::test::TestIOGraph *g, Type quantized_dtype,
```

Comment:
(optional) as this is another version of `quantize_and_verify_with_wrong_type` with explict `target` node as input parameter, it would be better to add a comment like
```
// Helper function to reduce duplicate test codes with a target as input parameter
```
as existing had a comment but this new one didn't

Reply:
I had thought about that, but I didn't put the line, as this verifier file is a test.cpp file. If a function is declared in a test file they are already a helper function, and looks redundant to tell :-D

Reply:
Also `reduce duplication` sounds kinda function's basic role, so...

Reply:
> already a helper function, and looks redundant to tell 

Sound reasonable :)